src/estimation/src/BerdyHelper.cpp

```diff
@@ -1366,6 +1401,35 @@ bool BerdyHelper::computeBerdySensorMatrices(SparseMatrix<iDynTree::ColumnMajor>
         // bY for the joint wrenches is zero
     }
 
+    ////////////////////////////////////////////////////////////////////////
+    ///// COM ACCELERATION SENSOR
+    ////////////////////////////////////////////////////////////////////////
+    if (m_options.includeCoMAccelerometerAsSensor)
+    {
+        // Get the row index corresponding to the com accelerometer sensor
+        IndexRange comAccelerometerRange = this->getRangeCoMAccelerometerSensorVariable(COM_ACCELEROMETER_SENSOR);
+
+        for(LinkIndex idx = 0; idx < static_cast<LinkIndex>(m_options.comConstraintLinkIndexVector.size()); idx++)
+        {
+            // Get the column index corresponding to the net link external wrench sensor
+            IndexRange netExternalWrenchSensor = this->getRangeLinkSensorVariable(NET_EXT_WRENCH_SENSOR,
+                                                                                  static_cast<LinkIndex>(m_options.comConstraintLinkIndexVector.at(idx)));
+
+            iDynTree::Rotation base_R_link = base_H_links(idx).getRotation();
+            iDynTree::Matrix3x3 base_R_link_M33;
+            iDynTree::toEigen(base_R_link_M33) = iDynTree::toEigen(base_R_link);
+
+            // Get link to base rotation
+            matrixYElements.addSubMatrix(comAccelerometerRange.offset,
+                                         netExternalWrenchSensor.offset,
+                                         base_R_link_M33);
```

Comment:
we are not filling the matrices with the mass information. So, as I understood the mass is supposed to be included in the COM acceleration vector.  If this is the case I think it should be specified, because it is not clear from the name (`COM_ACCELEROMETER_SENSOR `)